platforms/minigame/platforms/alipay/wrapper/fs-utils.js

```diff
@@ -22,6 +22,7 @@
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  THE SOFTWARE.
  ****************************************************************************/
+/* eslint-disable no-undef */
```

Comment:
Why need it?

Reply:
The global variable `my` is defined by minigame platform which couldn't be recognized.
 The eslint error is : 
```
'my' is not defined.eslint[no-undef](https://eslint.org/docs/latest/rules/no-undef)
```